build/setup.sql

```diff
@@ -81,49 +81,50 @@ insert into lead_providers(id, name) values
   (3, 'TF')
 ;
 
-create table tenureships (
+create table rosters (
   id integer primary key,
   participant_id integer references participants(id),
   school_id integer references schools(id),
+  role varchar(64),
   started_on date not null,
   finished_on date null
 );
 
-insert into tenureships(id, participant_id, school_id, started_on, finished_on) values
+insert into rosters(id, participant_id, school_id, role, started_on, finished_on) values
```

Comment:
Roster! I like it.

Reply:
Haha it just popped into my head, was thinking 'register' first but that didn't seem right